docs/manual/system/settings.de.md

```diff
@@ -325,5 +325,6 @@ Damit die Änderungen aktiv werden, muss am Ende der Anwendungs-Cache über den
 ```bash
 vendor/bin/contao-console cache:clear --env=prod --no-warmup
 ```
+Dazu muss man sich im Cantao Installationsverzeichnis befinden (das ist der Ordner, in dem `web`liegt).
```

Comment:
```suggestion
Dazu muss man sich im Contao Installationsverzeichnis befinden (das ist der Ordner, in dem `web` liegt).
```

Reply:
Ich finde, dass die Formulierung missverstanden werden kann. Ohne dem Hinweis in der Klammer reicht es ja auch.

Reply:
> Ich finde, dass die Formulierung missverstanden werden kann. Ohne dem Hinweis in der Klammer reicht es ja auch.

Genau. Mir wäre das auch lieber.